mon-entreprise/test/regressions/__snapshots__/simulations.jest.js.snap

```diff
@@ -166,45 +172,73 @@ exports[`calculate simulations-impot-société: bénéfices 4`] = `"[3000,0]"`;
 
 exports[`calculate simulations-impot-société: bénéfices 5`] = `"[51044,0]"`;
 
-exports[`calculate simulations-impot-société: bénéfices 6`] = `"[555044,0]"`;
+exports[`calculate simulations-impot-société: bénéfices 6`] = `
+"[555044,0]
+Notifications affichées : entreprise . chiffre d'affaires . franchise de TVA dépassée . notification"
```

Comment:
La notification sur la franchise de TVA s'affiche sur le simulateur IS, ce qui n'est pas voulu je pense.

![image](https://user-images.githubusercontent.com/1730702/114026751-e4f8b700-9876-11eb-846c-4db90fbad54c.png)